camel/societies/role_playing.py

```diff
@@ -219,6 +228,9 @@ def get_sys_message_info(
             user_role_name (str): The name of the role played by the user.
             sys_msg_generator (SystemMessageGenerator): A system message
                 generator for agents.
+            assistant_description (str, optional): The description of the
```

Comment:
@lightaime I do not like how the constructor grows to 20+ parameters. This break the cohesion. RolePlaying now is responsible for knowing about all the agents and their details. We do not want to create a god class here.

Reply:
Emmm, it would be an issue. At least, I think `sth_description` is necessary, as it is available to define the actions/personalities/rules of the agents in the future. Need to be discussed.

Reply:
I have explained in this [link](https://github.com/camel-ai/camel/pull/240#discussion_r1290562032), please review it thanks.